{{cookiecutter.repo_name}}/.github/workflows/{{cookiecutter._ci_name}}.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
         fail-fast: false
         matrix:
           os: [macOS-latest, ubuntu-latest, windows-latest]
-          python-version: {{ '${{ fromJSON(needs.environment-config.outputs.python-matrix) }}' }}
+          python-version: {% raw %}${{ fromJSON(fromJSON(needs.environment-config.outputs.python-matrix)) }}{% endraw %}
```

Comment:
This is ridiculous... why

Reply:
like I'm legit annoyed that it's working

Reply:
It's a bit odd -- the output of the action looks like it should be something like `'["3.9", "3.10"]'`, right? Maybe it gets double-stringed somewhere?

Reply:
let me check upstream that I didn't gufe it up

Reply:
yeah it's an upstream issue it seems, one sec